app/tests/unit/pages/project-revision/[projectRevision]/index.test.tsx

```diff
@@ -321,7 +321,7 @@ describe("The Create Project page", () => {
     act(() => userEvent.click(screen.queryByText("Discard Project Revision")));
 
     expect(
-      screen.getByText(/All changes made will be permanently deleted./)
+      screen.getByText(/All changes made will be deleted./)
```

Comment:
The wording here was approved by our design team, we shouldn't be changing it. Maybe make it configurable in the `DangerAlert` component?